study.md

```diff
@@ -67,5 +69,5 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+after the request sent to the router it send the information to the controller, the controller traslate the request into JSON and send the information to the model  (post, get, delete, patch) and then the model connect to the database and send back the information to the contoller as an object and the controller renders it to the browser
```

Comment:
The controller does not translate request information into JSON before invoking model methods.